File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/ItemFile.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class ItemFile {
    @JsonProperty("ContentType")
    public String contentType;
    @JsonProperty("FileContents")
    public String fileContents;
    @JsonProperty("FileDownloadName")
    public String fileDownloadName;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/ItemFile.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class ItemFile(
    @JsonProperty("ContentType")
    val contentType: String? = null,
    @JsonProperty("FileContents")
    val fileContents: String? = null,
    @JsonProperty("FileDownloadName")
    val fileDownloadName: String? = null
)
